ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -252,6 +254,7 @@ protected void registerWebSocketEndpoints(ServerContainer container) {
           )
       );
       final ObjectMapper mapper = getJsonMapper();
+      mapper.registerModule(new StructSerializationModule());
```

Comment:
This is redundant. getJsonMapper() already does this.

Reply:
Yes, refactored this as @big-andy-coates suggested.